bedrock/mozorg/templates/mozorg/about/manifesto.html

```diff
@@ -141,8 +141,8 @@ <h1 itemprop="name">{{ self.page_title() }}</h1>
                 <section>
                   <h4>{{ _('Learn more') }}</h4>
                   <ul class="resources">
-                    <li><a href="{{ url('privacy.privacy-day') }}">{{ _('See how Mozilla works to put your privacy first') }}</a></li>
-                    <li><a href="https://blog.mozilla.org/privacy/">{{ _('Read about developments in privacy and data safety') }}</a></li>
+                    <li><a href="{{ url('teach.smarton.index') }}">{{ _('See how Mozilla works to put your privacy first') }}</a></li>
+                    <li><a href="https://blog.mozilla.org/netpolicy/">{{ _('Read about developments in privacy and data safety') }}</a></li>
```

Comment:
Privacy blog is being absorbed by the net policy blog (https://bugzilla.mozilla.org/show_bug.cgi?id=1205091). The old URL will redirect, of course, but might as well update it here while I'm in the neighborhood.